lib/cypress/app_config.rb

```diff
@@ -41,5 +41,12 @@ def self.[]=(key, val)
         Rails.cache.delete('config_values')
       end
     end
+
+    def self.refresh
+      Rails.cache.delete('config_values')
```

Comment:
Rather than deleting the cache key here, you can just set `force: true` below, so line 47 would look like `Rails.cache.fetch('config_values', force: true) do`